internal/gensupport/resumable.go

```diff
@@ -256,6 +256,10 @@ func (rx *ResumableUpload) Upload(ctx context.Context) (resp *http.Response, err
 				rCtx, cancel = context.WithTimeout(ctx, rx.ChunkTransferTimeout)
 			}
 
+			if resp != nil && resp.Body != nil {
```

Comment:
Add a comment about why this is necessary and why it happens at this point in the loop.